linetools/isgm/io.py

```diff
@@ -41,3 +51,83 @@ def abssys_from_json(filename):
 
     # Return
     return abs_sys
+
+
+def read_joebvp(filename, coord, llist=None, specfile=None, chk_vel=False):
+    """
+    Parameters
+    ----------
+    filename : str
+      joeB VP filename
+    coord : SkyCoord
```

Comment:
Explain coord is supposed to be the coordinate of the QSO sightline for the `AbsComponents`...

Reply:
added